artemis-unit-test-support/src/main/java/org/apache/activemq/artemis/utils/ThreadLeakCheckRule.java

```diff
@@ -244,7 +244,7 @@ private boolean isExpectedThread(Thread thread) {
          return true;
       } else if (threadName.contains("Attach Listener")) {
          return true;
-      } else if ((javaVendor.contains("IBM") || isSystemThread) && threadName.equals("process reaper")) {
+      } else if ((javaVendor.contains("IBM") || isSystemThread) && (threadName.equals("process reaper") || threadName.startsWith("process reaper (pid"))) {
```

Comment:
did you mean the (pid) in the string?

Reply:
yes... I think you just do startsWith("process reaper");


As the change on the idk is 

process reaper (pid " + pid + ")");

Reply:
It isnt always in the name, it comes and goes.

Reply:
But yes we could use 'starts with' for both if you prefer.